src/js/components/Calendar/StyledCalendar.js

```diff
@@ -61,35 +73,50 @@ const slideStyle = props => {
   `;
 };
 
+const weeksSizeStyle = props => {
+  return css`
+    ${props.fillContainer && 'display: flex;'}
+    ${props.fillContainer && 'flex-direction: column;'}
+    ${props.fillContainer && 'height: 100%;'}
+  `;
+};
 const StyledWeeks = styled.div`
   position: relative;
+  ${props => weeksSizeStyle(props)}
   ${props => props.slide && slideStyle(props)};
 `;
 
 StyledWeeks.defaultProps = {};
 Object.setPrototypeOf(StyledWeeks.defaultProps, defaultProps);
 
+const weekSizeStyle = props => {
+  return css`
+    display: flex;
+    justify-content: space-between;
+    ${props.fillContainer && 'flex: 1;'}
+  `;
+};
 const StyledWeek = styled.div`
-  display: flex;
-  flex-direction: row;
-  flex-justify: between;
+  ${props => weekSizeStyle(props)}
 `;
 
 StyledWeek.defaultProps = {};
 Object.setPrototypeOf(StyledWeek.defaultProps, defaultProps);
 
 const StyledDayContainer = styled.div`
   flex: 0 0 auto;
+  ${props => props.fillContainer && 'width: 14.3%;'}
```

Comment:
Can we add a comment here about why "14.3%", that it is 100/7?

Reply:
Sounds good. Update made.